advisor_alert.rst

```diff
@@ -0,0 +1,29 @@
+Advisor Alerts
+==============
+The Advisor is an entity in Scylla Monitoring that aims to find potential users' problems and advise on how to solve them.
+Currently, it uses low-priority alerts as a notification method.
+That means that for each potential issue, it will generate an alert.
+
+Following is a list of such alerts, with their description break into sections.
+
+CQL Optimization
+^^^^^^^^^^^^^^^^
+- cqlNonPrepared:Some queries are non-prepared
+- cqlNonPaged:Some SELECT queries  are non-paged
+- cqlNoTokenAware:Some queries are not token-aware
+- cqlReverseOrder:Some queries use reverse order
+- cqlAllowFiltering:Some queries use ALLOW FILTERING
+- cqlCLAny:Some queries use Consistency Level: ANY
+- cqlCLAll:Some queries use Consistency Level: ALL
+
+
+Balanced
+^^^^^^^^
+- nonBalancedcqlTraffic:CQL queries are not balanced among shards
+
+Ooperation Error
+^^^^^^^^^^^^^^^^
```

Comment:
Advisor alert for selected errors in the cluster

Reply:
Does this alert base on logs? if so we need to mention it, as most user will need use Loki at first

Reply:
All those are not, but I'll add a section for loki's alerts